website/docs/faq.md

```diff
@@ -0,0 +1,65 @@
+---
+id: faq
+title: FAQ
+---
+
+A collection of frequently asked questions.
+
+## Billing, Payments & Subscriptions
+
+### What if I exceed the [config.json download](requests) limit of my plan?
+Don't worry, we will keep serving your data and feature flags. Someone from our team will contact you to discuss your options.
+You can always check your Usage & Quota [here](https://app.configcat.com/organization/usage).
+
+### Where can I find and download my invoices?
+All the invoices we issued are available for download from the [Billing & Invoices page.](https://app.configcat.com/organization/billing).
+You need to have a [Billing Manager](organization#billing-manager-role) role to access the Billing & Invoices page.
+
+### How do I upgrade / downgrade my billing plan?
+You can change your billing plan on the [Plans](https://app.configcat.com/organization/plans) page.
+You need to have a [Billing Manager](organization#billing-manager-role) role to access the Plans page.
+
+### How do I change my payment method or billing information?
+Go to the [Billing & Invoices page.](https://app.configcat.com/organization/billing) and click the `Update billing details` link.
+You need to have a [Billing Manager](organization#billing-manager-role) role to access the Billing & Invoices page.
+
+### How do I cancel my account?
+Go to the [Plans](https://app.configcat.com/organization/plans) page and click the `Switch Plan` button under the Free plan.
+You need to have a [Billing Manager](organization#billing-manager-role) role to access the Plans page.
+
+### How do I change my currency?
+Go to the [Plans](https://app.configcat.com/organization/plans) page and use the currency toggle to switch between USD and EUR.
+You need to have a [Billing Manager](organization#billing-manager-role) role to access the Plans page.
+
+## Security
+### I can't login to ConfigCat, because I was asked a 2FA (Two-factor authentication) code that I don't know.
+
+*Solution 1:* There might be an authenticator app on your phone that you can use to login to ConfigCat.
+
+*Solution 2:* Contact your `Organization Admin`, and ask them to disable 2FA for your account until you set it up again.
+
+*Solution 3:* Use your recovery codes that you received when you first set your 2FA up.
+
+## A/B Testing & Targeting
+
+### How do I use AND operators in my targeting rules?
+Although there is no direct support for `AND` operators in the UI, you can use the `AND` operator as follows:
+
+🤓 Via [De Morgan's laws](https://en.wikipedia.org/wiki/De_Morgan%27s_laws): `Condtion1 AND Condition2` is equivalent to `NOT (NOT Condition1 OR  NOT Condition2)`.
```

Comment:
Kicsit talán jobban érthető tényleg Zoli magyarázata, ConfigCat-es rule-okra könnyebben rá lehet húzni

IF Condition1 AND Condition2 THEN True
is equivalent to
IF NOT Condition1 THEN False
IF NOT Condition2 THEN False
ELSE TRUE